examples/simple-udp/server/main.go

```diff
@@ -50,7 +49,7 @@ func main() { // nolint: gocyclo
 	}
 
 	log.Print("Starting Health Ping")
-	stop := make(chan bool)
+	stop := make(chan struct{})
```

Comment:
Just for my own curiosity - why the change? Because it more closely matches `context.Context` and some of the Kubernetes constructs?

Reply:
Since `stop` is used purely as a signally mechanism, and not as a way to transport meaningful information, I have followed the Go idiom of using `struct{}`, which indicates the value is of no meaning in and of itself. On the other hand, `bool` suggests we might send `true` or `false`, when in fact we don't really care about either.

Reply:
Thanks! That makes perfect sense!